/bin/sh: 1: /app/run_etl_periodico.sh: Permission denied
/app/run_etl_periodico.sh: line 8: python: command not found
ETL ejecutado a 2025-06-02T20:35:36Z
ETL ejecutado a 2025-06-02T21:04:29Z
ETL ejecutado a 2025-06-02T21:34:17Z
ETL ejecutado a 2025-06-02T22:04:18Z
